spec/models/validations/sales/setup_validations_spec.rb

```diff
@@ -26,8 +21,8 @@
         end
       end
 
-      context "when saledate is in the 22/23 collection year" do
-        let(:record) { build(:sales_log, saledate: Time.zone.local(2023, 1, 1)) }
+      context "when saledate is in the open collection year" do
+        let(:record) { build(:sales_log) }
```

Comment:
with the description in the context block here I would expect we would still need a saledate?
`:saledate_today` trait?

Reply:
You're right, I thought that was the default!